### PR TITLE
Chore: Fix spacing issues in Explore page and List-Item-Header

### PR DESCRIPTION
**Pull request by RakeshUP**
Task: [APP-1504](https://aragonassociation.atlassian.net/browse/APP-1504), [APP-1506](https://aragonassociation.atlassian.net/browse/APP-1506)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1504]: https://aragonassociation.atlassian.net/browse/APP-1504?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-1506]: https://aragonassociation.atlassian.net/browse/APP-1506?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sepehr2github — APPROVED** [review]
Thanks

File: packages/web-app/src/containers/carousel/index.tsx [hunk 1]
```diff
@@ -116,7 +116,7 @@ const DesktopCTA = styled.div.attrs({
 })``;
 
 const MobileCTA = styled.div.attrs({
-  className: 'relative -mt-15 mb-5 -mx-2 tablet:-mx-3 desktop:mx-0',
+  className: 'relative -mt-13 mb-5 -mx-2 tablet:-mx-3 desktop:mx-0',
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/carousel/index.tsx:119]: This doesn't quite correspond to the designs :(

With this, the carousel is completely below the colored section, while it should be half on top of the colored section and half on top of the white one.

**Review by WissenIstNacht — CHANGES_REQUESTED** [review]

File: packages/web-app/src/containers/carousel/index.tsx (hunk 1 above)
Comment (RakeshUP) [reply packages/web-app/src/containers/carousel/index.tsx:119]: Thanks for catching that. I think my hot reload didn't work properly when I made that change. And my placebo made me believe that the card had actually moved by 16px! :)

File: packages/ui-components/src/components/button/buttonBase.tsx [hunk 2]
```diff
@@ -51,9 +51,9 @@ const sizeClassNames = {
 };
 
 const sizeStyles = {
-  small: {minWidth: '32px'},
-  medium: {minWidth: '40px'},
-  large: {minWidth: '48px'},
+  small: {minWidth: '64px'},
```
Comment (WissenIstNacht) [inline packages/ui-components/src/components/button/buttonBase.tsx]: This is not really feasible :/

This makes some buttons in the app too wide. E.g., icons and pagination.

**Review by WissenIstNacht — CHANGES_REQUESTED** [review]
I'm really sorry for the situation with the button ticket, Rakesh.

Nevertheless, the current changes can't be merged in. I would suggest taking those changes out and just proceed with the other two tickets.

And thanks for addressing the comments 👍🏼

**Review by WissenIstNacht — APPROVED** [review]

**Pull request merged**